varats/paper/case_study.py

```diff
@@ -119,8 +117,20 @@ def sort(self, reverse: bool = True) -> None:
         """
         self.__revisions.sort(key=lambda x: x.commit_id, reverse=reverse)
 
+    def get_dict(self) -> tp.Dict[
+        str, tp.Union[str, tp.List[tp.Dict[str, tp.Union[str, int]]]]]:
```

Comment:
```suggestion
            self
    ) -> tp.Dict[str, tp.Union[str, tp.List[tp.Dict[str, tp.
                                                    Union[str, int]]]]]:
```
line continuation should have more indentation as the prevs line

Reply:
Resolved in 4cd506d